### ISSUE TITLE
Export all API calls to their own service (36 ⛔)

### ISSUE THREAD

**Issue by OPerel** [wonderfloyd/Hackathon-Corona#45]
_Currnet:_ API calls (at the moment only to Contentful) are made from the pages.
_Expected:_ API calls should be made from a designated service and imported where needed.

__Note: this is blocked by https://github.com/wonderfloyd/Hackathon-Corona/issues/36 because working in it will result in more API calls to contentful__

**Comment by orc886** [comment]
I was thinking about the best way to implement this and what I believe we should do is add a button in the post page which allows applying the generated excerpt to the post
This way we can add a new task and 36 will not block it
